### PR TITLE
Pull NLP additions

### PR DESCRIPTION
**Pull request by fmatthies**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	<groupId>care.smith.top</groupId>
 	<artifactId>top-api</artifactId>
-	<version>0.8.0</version>
+	<version>0.8.0-SNAPSHOT</version>
```
Comment (ChristophB) [inline pom.xml:7]:
```suggestion
	<version>0.9.0</version>
```

Comment (ChristophB) [reply pom.xml:7]: Please also modify the version number in the schema file.

File: schemas/top-api.yml [hunk 2]
```diff
@@ -122,6 +104,11 @@ paths:
           description: Filter documents by name.
           schema:
             type: string
+        - name: exemplarOnly
```
Comment (ChristophB) [inline schemas/top-api.yml:107]: What is the purpose of this query parameter?

**Review by ChristophB — CHANGES_REQUESTED** [review]
Please also perform `npx openapi-format schemas/top-api.yml --output schemas/top-api.yml` to reformat the schema file. In future, this step will be added as a GitHub workflow.

File: schemas/top-api.yml [hunk 3]
```diff
@@ -94,24 +81,19 @@ paths:
...
-      operationId: getDocumentsByConceptIds
+      operationId: getDocumentIdsByConceptClusterIds
       summary: Searches for documents from specific concept clusters. Use the query parameters to filter the result.
       parameters:
-        - name: idOnly
-          in: query
-          schema:
-            type: boolean
-            default: true
         - name: gatheringMode
```
Comment (ChristophB) [inline schemas/top-api.yml:97]: Add a short description. Mention possible values.

**Comment by fmatthies** [comment]
> Please also perform `npx openapi-format schemas/top-api.yml --output schemas/top-api.yml` to reformat the schema file. In future, this step will be added as a GitHub workflow.

Done, but it didn't change anything.

**Review by fmatthies — COMMENTED** [review]
applied requested changes

**Review by ChristophB — APPROVED** [review]

**Pull request merged**